include/pros/rtos.hpp

```diff
@@ -279,6 +279,17 @@ class Task {
 	 */
 	std::uint32_t notify();
 
+	/**
+	 * Waits until specified task is complete and deleted, then continues to execute the program.
+	 * Analogous to std::thread::join in C++.
+	 *
+	 * See https://pros.cs.purdue.edu/v5/tutorials/topical/notifications.html for
+	 * details.
+	 *
+	 * \return void
+	 */
+	std::void_t join();
```

Comment:
std::void_t is definitely not correct here.

Reply:
wait then what is (i am not familiar with c++)